objects/aidBoard/script.lua

```diff
@@ -1043,17 +1043,22 @@ end
 function scanElements()
     local elements = Elements:new()
     for _, selected in pairs(Global.getVar("selectedColors")) do
-        elements:add(selected.nonTokenElements)
+        elements:add(selected.elementCounts)
     end
 
     local elementsTable = {"Sun","Moon","Fire","Air","Water","Earth","Plant","Animal"}
     for i,total in ipairs(elements) do
-        local elementTokensCount = #getObjectsWithTag(elementsTable[i])
+        for _,obj in pairs(getObjectsWithTag(elementsTable[i])) do
+            -- Don't count markers in player areas
```

Comment:
why are you skipping element tokens in player areas?

Reply:
The first commit in this PR was to change how the aid board avoids double-counting token elements, as preparation for ensuring that the hook would actually work properly. This is explanation I gave in 54fbe13085f56b1431bae23553134833b3bbfc58's commit message:

To avoid double-counting element markers in player areas for the aid board, each player area's `countItems()` loop counts non-token elements separately. This makes it difficult to implement a hook to modify elements, as both counting methods, `countItems()` and `aidBoard`'s `scanElements()`, have to be handled separately. To make this easier, shift the onus for avoiding double-counting to the aid board, removing the need for tracking non-token elements. This likely comes with a performance penalty, as now `scanElements()` must call `getZones()` on each element token. But `scanElements()` runs only every two seconds, and `getZones()` isn't a raycast, so this shouldn't be too bad.

Reply:
gotcha so `selected.elementCounts` will handle everything in player zones (track elements, power cards, and tokens) so the job here is to count element tokens not in player area, since folks often pull them out to keep track of how much folks have paid for an event

Reply:
Yep. And not just ones that people pull out for events: this also counts ones from Elemental Invocation.